src/portable/nordic/nrf5x/dcd_nrf5x.c

```diff
@@ -361,8 +454,9 @@ bool dcd_edpt_xfer (uint8_t rhport, uint8_t ep_addr, uint8_t * buffer, uint16_t
 void dcd_edpt_stall (uint8_t rhport, uint8_t ep_addr)
 {
   (void) rhport;
+  uint8_t const epnum = tu_edpt_number(ep_addr);
 
-  if ( tu_edpt_number(ep_addr) == 0 )
+  if ( epnum != 0 && epnum != 8 )
```

Comment:
the if logic condition is inverted, the condition here is for control endpoint

Reply:
my bad, thanks for spotting it

Reply:
no problem, my original code is not too consistent with if else between stall and clear stall. I must write it at 2 different time and it is easy to overlook.